### PR TITLE
SOT-69: Write Chapter5 of Proofs, Args, and Zk

### PR DESCRIPTION
**Pull request by VanhGer**
# Check List

- [x] Did you check your grammar / reader proof your document using ChatGPT?
- [x] Don't forget to squash commits into meaningful chunks before merging
- [x] After merging this, please update your task status!
- [x] Did you verify that your math formulas are rendered correctly?

### CODE REVIEW

File: terms/correlation_intractability.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Correlation-Intractability (CI)
+
+Let $R$ denote some property of pairs $(y,h(y))$. A hash family $H$ satisfies **CI** for $R$ if it is computationally infeasible to
+find a pair $(y, h(y))$ satisfying property $R$.
+
+Suppose $I$ is an [IP](ip.md) for a [language](language.md) $L$ such that $I$ satisfies
+[round-by-round soundness](../articles/proofs-arguments-and-zero-knowledge/chapter_5.md#Round-by-round%20soundness%20requirements).\
```
Comment (ktokto313) [inline terms/correlation_intractability.md:7]: where is the round-by-round soundness requirement.md?

**Review by ktokto313 — APPROVED** [review]

File: terms/correlation_intractability.md (hunk 1 above)
Comment (VanhGer) [reply terms/correlation_intractability.md:7]: ok sure I'll add this file soon.

**Review by ktokto313 — APPROVED** [review]

**Review by Tranduy1dol — APPROVED** [review]

File: articles/proofs-arguments-and-zero-knowledge/chapter_5.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- [Section 9.2](chapter_9.md): Fiat-Shamir transformation preserves knowledge-soundness when apply to succinct arguments obtained
+  from PCPs and IOPs.
+- [Section 12.2](chapter12.md): The transformation is applied to $\sum$-protocols.
+
+## Fiat-Shamir in the Plain Model
+
+When the concrete hash function $h$ is chosen at random from a hash family $H$ satisfying
+[CI](../../terms/correlation_intractability.md), instantiating the [Fiat-Shamir](../../terms/fiat_shamir.md) transformation in the
+plain model results in a sound
+argument .
```
Comment (Draply) [inline articles/proofs-arguments-and-zero-knowledge/chapter_5.md:78]: remove space.

File: terms/fiat_shamir.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+
+Fiat-Shamir is often implemented using a technique called: ==hash chaining==. This means that, instead of calculating $r_i$ by hashing
+all the preceding prover messages: $g_1,...g_i$, it chooses $r_i$ to be hash only of $(x, i, r_{i-1}, g_i)$.
+
+## Avoiding a Common Vulnerability
+
+For the Fiat-Shamir transformation to be secure in settings where an adversary can choose the input $x$ to the [IP](ip.md) or
+[argument](arguments.md), it is essential that $x$ be appended to the list that is hashed in each round.
+
+For example, in the [GKR protocol](gkr_protocol.md), the verifier $V$ completely ignores the input $x \in F^n$ until the final check
```
Comment (haipro287) [inline terms/fiat_shamir.md:32]: I think this should be $\mathbb{F}$. Please check again all docs

**Review by Draply — APPROVED** [review]

**Pull request merged**